sw/device/tests/penetrationtests/firmware/ibex_fi.c

```diff
@@ -586,6 +586,111 @@ status_t handle_ibex_fi_char_sram_read(ujson_t *uj) {
   return OK_STATUS();
 }
 
+status_t handle_ibex_fi_char_sram_write_static(ujson_t *uj) {
+  // Clear registered alerts in alert handler.
+  sca_registered_alerts_t reg_alerts = sca_get_triggered_alerts();
+
+  // Initialize SRAM region with inverse ref_values to avoid that data from a
+  // previous run is still in memory.
+  for (size_t i = 0; i < 64; i++) {
+    sram_main_buffer[i] = ~ref_values[0];
```

Comment:
Alternatively we could also use `memset` here I think? even simpler. Only works if the buffer is initialized all to the same value.